internal/driver/config/opl_config_namespace_watcher.go

```diff
@@ -30,15 +35,45 @@ type (
 var _ namespace.Manager = (*oplConfigWatcher)(nil)
 
 func newOPLConfigWatcher(ctx context.Context, l *logrusx.Logger, target string) (*oplConfigWatcher, error) {
-
 	nw := &oplConfigWatcher{
 		logger:                 l,
 		target:                 target,
 		files:                  configFiles{byPath: make(map[string]io.Reader)},
 		memoryNamespaceManager: *NewMemoryNamespaceManager(),
 	}
 
-	return nw, watchTarget(ctx, target, nw, l)
+	targetUrl, err := urlx.Parse(target)
```

Comment:
Please use ory/x/fetcher instead, it can load from all sources and has lots of tests.

Reply:
done